blockbuster/Cargo.toml

```diff
@@ -11,10 +11,14 @@ readme = "../README.md"
 [dependencies]
 bytemuck = { version = "1.14.0", features = ["derive"] }
 spl-token-2022 = { version = "1.0", features = ["no-entrypoint"] }
-spl-account-compression = { version = "0.3.0", features = ["no-entrypoint"] }
-spl-noop = { version = "0.2.0", features = ["no-entrypoint"] }
-mpl-bubblegum = "1.2.0"
-mpl-core = { version = "0.5.0", features = ["serde"] }
+
+# Update these deps to original SPL crate after merging 'feature/init_with_root' changes into original repo.
+spl-account-compression = { git = "https://github.com/StanChe/solana-program-library.git", branch = "feature/init_with_root", features = ["no-entrypoint"] }
+spl-noop = { git = "https://github.com/StanChe/solana-program-library.git", branch = "feature/init_with_root", features = ["no-entrypoint"] }
+mpl-bubblegum = { git = "https://github.com/adm-metaex/mpl-bubblegum", branch = "feature/cleanup", features = ["test-sbf", "serde"] }
+# ^^^
+
+mpl-core = { version = "=0.8.0-beta.1", features = ["serde"] }
```

Comment:
Why beta? Can't we use a stable version?

Reply:
Reference implementation use the same version for  
mpl-core